# Parameter msg.

string param_id
mavros_msgs/ParamValue value
uint8 param_type
uint16 param_count
uint16 param_index
